### PR TITLE
Bug/stackblitz icons

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- This fixes the icons not loading in stackblitz and removes duplicate dividers.

https://stackblitz.com/github/pxblue/angular-showcase-demo/tree/bug/stackblitz-icons

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -58,7 +58,11 @@ export class AppComponent {
         this.colors = PXBColors;
         this._matIconRegistry.addSvgIconSetInNamespace(
             'px-icons',
-            this._domSanitizer.bypassSecurityTrustResourceUrl(iconSet)
+            // Stackblitz works with https request instead of using icons from node_modules.
+            this._domSanitizer.bypassSecurityTrustResourceUrl(
```
Comment (joebochill) [inline src/app/app.component.ts:62]: Hmm. Looks like another situation liked we talked about for the workflows demo images:

"Do we do things that we don't recommend adopters do for the sake of having the demo run in <StackBlitz | CodeSandbox | Snack>?"

Comment (emclaug2) [reply src/app/app.component.ts:62]: I think we shouldn't make our requirements the exact same as our adopters; no one is going to have to host their finished product in a third-party container like sandbox or stackblitz. 

I think if we add a comment explaining the situation and what adopters should normally do here, I think we should be fine.

Comment (joebochill) [reply src/app/app.component.ts:62]: That's fine. If we include both options and comment out the correct one and put in a biiiig comment that explains how to switch it back, we should be fine.

**Review by joebochill — APPROVED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (emclaug2) [reply src/app/app.component.ts:66]: Ok cool, I added a pretty big comment explaining the situation to our end users.  I think this should be fine.  It includes the code snippets required to normally import the iconSet.

**Pull request merged**